### DISCUSSION TITLE
Understanding Kit: What's its purpose?

### DISCUSSION THREAD

**Discussion by benaltair** [sveltejs/kit#2728] [Q&A] [answered]
Hey all, so basic question here: what is the vision for Kit 1.0 in the Svelte ecosystem? I've been using it for a few projects, so I know what it is (that is: a router, Vite for local dev server and SSR, certain accompanying configs), but I'm trying to understand how that fits into the wider Svelte plans. 

Some have compared Kit with `create-react-app` - is that accurate? Is this basically an opinionated preconfigured setup for Svelte itself? This seems to be the case, especially with `npm init svelte` being the future command to set up a Kit project. 

Curious on the community's thoughts

**Comment by CaptainCodeman** [comment, marked as answer]
It's the Application Framework to Svelte's Component Framework.

In a similar way that Next.js builds on top of React to provide the higher level functionality that you need to build a complete app rather than individual components.

Things like routing, session handling, pre-fetching, and the ability to support a continuous spectrum of app types from SSR to SPA, including hybrid.

**Reply by benaltair** [reply]
That makes sense. Thanks

**Reply by tropicaaal** [reply]
Also to answer your question about `create-react-app`, that's more analgous to the starter template that you are linked to in the initial svelte tutorial. Both are just basic SPAs with more or less the bare minimum to start using the framework. Essentially, Svelte is the component framework and SvelteKit is the application framework.

**Reply by amartens181** [reply]
How would kit fit into situations where we only want route management? There's the static adapter, but there is no debug flow for that. 
I basically want .Net to handle guarding the web app and providing APIs and have Svelte provide the UI. Is that overkill for kit?

**Reply by benmccann** [reply]
Even if you want to write your backend in another language (as I normally do myself), it doesn't necessarily mean you have to use `adapter-static`. You can still do SSR in that case if you'd like
